app/Models/Training/WaitingList.php

```diff
@@ -88,6 +89,17 @@ public function flags()
         return $this->hasMany(WaitingListFlag::class, 'list_id');
     }
 
+    /**
+     * Scope a query to only include ATC waiting lists.
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function scopeAtc($query)
+    {
+        return $query->where('department', '=', 'atc');
```

Comment:
Seeing as we have this defined as a ENUM constant up top, we should use that for easier refactoring in future / single-source-of truth
```suggestion
        return $query->where('department', '=', self::ATC_DEPARTMENT);
```